assets/scripts/views/actions.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * The floating actions
+ */
+export default class {
+  /**
+   * Initialize
+   */
+  constructor({
+    onMenuToggle = () => {},
+    onFiltersToggle = () => {},
+    onCenter = () => {}
+  }) {
```

Comment:
Are these default handlers, that will be overridden? It looks a bit strange here, cause of the fat arrow syntax.